docs/core/additional-tools/dotnet-core-uninstall.md

```diff
@@ -0,0 +1,339 @@
+
+__Note__: The tool requires elevation to uninstall .NET Core SDKs and Runtimes. Therefore, it should be installed in a write-protected directory such as `C:\Program Files` on Windows and `/usr/local/bin` on macOS. See also [Elevated access for dotnet commands](https://docs.microsoft.com/dotnet/core/tools/elevated-access).
+
+### Displaying installed .NET Core SDKs and Runtimes
+
+`dotnet-core-uninstall list` lists installed .NET Core SDKs and Runtimes that can be removed with this tool.
+
+On Windows, the .NET Core Uninstall Tool can only uninstall things that have been installed via the .NET Core SDK or Runtime installer, or the Visual Studio installer prior to Visual Studio 2019 16.3. On macOS, the tool can only uninstall things in the `/usr/local/share/dotnet` folder. This tool does not work on Linux. Because of these limitations, the tool may not be able to uninstall all of the .NET Core SDKs and Runtimes on your machine. You can use `dotnet --info` to find all of the .NET Core SDK's and Runtimes available, including those this tool cannot remove.
+
+#### Synopsis
+
+```
+dotnet-core-uninstall list [options]
+```
+
+#### Options
+
+# [Windows](#tab/windows)
+
+* **`--aspnet-runtime`**
+
+  List ASP.NET Core Runtimes.
+
+* **`--hosting-bundle`**
+
+  List .NET Core Runtime & Hosting Bundles.
+
+* **`--runtime`**
+
+  List .NET Core Runtimes.
+
+* **`--sdk`**
+
+  List .NET Core SDKs.
+
+* **`-v, --verbosity <LEVEL>`**
+
+  Set the verbosity level. Allowed values are `q[uiet]`, `m[inimal]`, `n[ormal]`, `d[etailed]`, and `diag[nostic]`.
+
+* **`--x64`**
+
+  List x64 .NET Core SDKs or Runtimes.
+
+* **`--x86`**
+
+  List x86 .NET Core SDKs or Runtimes.
+
+# [macOS](#tab/macos)
+
+* **`--runtime`**
+
+  List .NET Core Runtimes.
+
+* **`--sdk`**
+
+  List .NET Core SDKs.
+
+* **`-v, --verbosity <LEVEL>`**
+
+  Set the verbosity level. Allowed values are `q[uiet]`, `m[inimal]`, `n[ormal]`, `d[etailed]`, and `diag[nostic]`.
+
+* **`--x64`**
+
+  List x64 .NET Core SDKs or Runtimes.
+
+* **`--x86`**
+
+  List x86 .NET Core SDKs or Runtimes.
+  
+---
+
+#### Examples
+
+* List all .NET Core SDKs and Runtimes that can be removed with this tool:
+
+  ```
+  dotnet-core-uninstall list
+  ```
+
+* List all x64 .NET Core SDKs and Runtimes:
+
+  ```
+  dotnet-core-uninstall list --x64
+  ```
+
+* List all x86 .NET Core SDKs:
+
+  ```
+  dotnet-core-uninstall list --sdk --x86
+  ```
+
+### Uninstalling .NET Core SDKs and Runtimes
+
+`dotnet-core-uninstall` uses a collection of options to specify what will be uninstalled.
+
+Since this is a destructive tool, it is **highly** recommended that you run with the `--dry-run` option first to determine the .NET Core SDKs and Runtimes that will be removed. [This article explains which SDKs and Runtimes are safe to remove.](https://docs.microsoft.com/dotnet/core/versions/remove-runtime-sdk-versions#should-i-remove-a-version)
```

Comment:
```suggestion
Since this is a destructive tool, it is **highly** recommended that you run with the `--dry-run` option first to determine the .NET Core SDKs and Runtimes that will be removed. [This article explains which SDKs and Runtimes are safe to remove.](../versions/remove-runtime-sdk-versions#should-i-remove-a-version)
```